docs/blocks.md

```diff
@@ -20,24 +20,25 @@ The block is an living standard and blocks go through a life-cycle as mentioned
 
 ## List
 
-Name |Stability| What can you do with it? | Added
--- | -- | -- | -- |
+Name |Stability| What can you do with it? | Added | Last updated
+-- | -- | -- | -- | -- |
 [Start](#Start) | Experimental | Starts the flow program | 21.09.1
 [Send Request](#Send-Request) | Experimental | Executes the selected request | 21.09.1
 [Terminal](#Terminal)| Experimental | Displays data that is incoming to the block | 21.09.1
-[Assign Variables](#Assign-Variable)| Experimental | Assign strings to template variables
-[Create Variable](#Create-Variable) | Experimental | Assign data from reference to template variables
 [Delay](#Delay) | Experimental |Adds a delay | 21.09.1
 [Test Summary](#Test-Summary) | Experimental | Fetches test summary of the request and displays it in terminal fashion
-[Validate](#Validate)| Experimental | Checks if the query matches certain data or not | 21.09.1
 [For Each](#For-Each) | Experimental | Iterate over list | 21.09.1
 [Merge](#Merge) | Experimental |Takes 2 objects and merges them into a single object 
 [List Pop](#List-Pop) | Experimental | Get element at index 0 of an array 
-[Group By](#Group-By) | Experimental |Groups an array based on a specified key 
-[Parse JSON](#Parse-JSON) | Experimental |Parses the incoming data to JSON | 21.09.1
+[Group By](#Group-By) | Experimental | Groups an array based on a specified key 
+[Parse JSON](#Parse-JSON) | Removed |Parses the incoming data to JSON | 21.09.1
 [Concatenate](#Concatenate) | Experimental | Take 2 data pieces and put them in a single array list
-[Condition](#Condition) | Experimental | Checks for the given condition and passes data to either "Accept" or "Reject" port
-[Check](#Check) | Experimental | Check for the given condition from 2 different inputs and pass data only if condition resolves to `True`
+[Condition](#Condition) | Experimental | Checks for the given condition and passes data to either "Accept" or "Reject" port | 21.11.1
+[Check](#Check) | Experimental | Check for the given condition from 2 different inputs and pass data only if condition resolves to `True` | 21.11.1
+[Create Durables](#Create-Durables) | Experimental | Set some value to be made durable | 21.12.2 | 21.12.2
+[Assign Variables](#Assign-Variable)| Deprecated | Use Send Request's configuration to pass data to request variables |  | 21.12.2
```

Comment:
What was updated in 21.12.2?

Reply:
Adding each and every change that we make to a block, will make things clumsy in this file.

Reply:
Where else is the source of truth?